src/Magento/FunctionalTestingFramework/Codeception/Subscriber/Console.php

```diff
@@ -31,6 +31,18 @@ class Console extends \Codeception\Subscriber\Console
      */
     private $actionGroupStepKey = null;
 
+    /**
+     * Console constructor. Feeds the second option passed into parent constructor.
```

Comment:
Can you add a short description explaining why the `$extensionOptions` variable is not passed to the parent constructor?